project.godot

```diff
@@ -14,6 +14,10 @@ config/name="IT23-Game"
 config/features=PackedStringArray("4.2", "GL Compatibility")
 config/icon="res://icon.svg"
 
+[autoload]
+
+Mobtarget="*res://Mobtarget.gd"
```

Comment:
I'm not happy about this name. I think it should be something more general so we can put more stuff into it in the future.

Reply:
will be fixed for next commit